File: js/capitalizeTheNames.js
// Create a function that takes an array of names and returns an array where only
// the first letter of each name is capitalized.
// https://edabit.com/challenge/ZN5cpGPNRS3nrjMo5

const transform = (item) =>
  `${item.substring(0, 1).toUpperCase()}${item.substring(1).toLowerCase()}`;

const capMe = (arr) => arr.map(transform);

module.exports = capMe;

File: CSharp/CapitalizeTheNames.cs
using System.Linq;

namespace CSharp
{
    // Create a function that takes an array of names and returns an array where only 
    // the first letter of each name is capitalized.
    // https://edabit.com/challenge/AafHF2YMGCqDRWERo
    public static class CapitalizeTheNames
    {
        public static string[] CapMe(string[] arr) =>
            arr.Select(name => $"{name.Substring(0, 1).ToUpper()}{name.Substring(1).ToLower()}").ToArray();
    }
}